src/main/java/pl/touk/service/DriverService.java

```diff
@@ -0,0 +1,66 @@
+package pl.touk.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import pl.touk.model.Driver;
+import pl.touk.repository.DriverRepository;
+import pl.touk.service.dto.NewDriverConverter;
+import pl.touk.service.dto.NewDriverDto;
+import pl.touk.service.dto.UpdateDriverConverter;
+import pl.touk.service.dto.UpdateDriverDto;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DriverService {
+
+    private final DriverRepository driverRepository;
+    private final NewDriverConverter newDriverConverter;
+    private final UpdateDriverConverter updateDriverConverter;
+    private final CurrentTimeService currentTimeService;
+    private final ParkingRateService parkingRateService;
+
+    public NewDriverDto startParkingMeter(String registrationNumber) {
+        if (StringUtils.isEmpty(registrationNumber)) {
+            throw new IllegalArgumentException("Registration number is required");
+        }
+        Driver driver = new Driver();
+        driver.setStartParkTime(currentTimeService.now());
```

Comment:
czemu nie ustawić tych wartości w konstruktorze?

Reply:
Można to zrobić przez konstruktor i wsumie jest to bardziej czytelne.